### PR TITLE
[CPU] [ARM64] jit clamp

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [AARCH64] jit eltwise Clamp

### Tickets:
 - *CVS-133829*

### CODE REVIEW

File: src/core/src/op/multiply.cpp [hunk 1]
```diff
@@ -51,7 +51,7 @@ bool Multiply::evaluate(TensorVector& outputs, const TensorVector& inputs) const
                                       this,
                                       outputs,
                                       inputs,
-                                      OV_PP_ET_LIST(f32, f64, i32, i64, u32, u64),
+                                      OV_PP_ET_LIST(f16, f32, f64, i8, i32, i64, u8, u32, u64),
```
Comment (praasz) [inline src/core/src/op/multiply.cpp:54]: Why this change is required?

**Comment by dmitry-gorokhov** [comment]
@xuchen-intel Please review the PR

**Review by xuchen-intel — APPROVED** [review]
LGTM with some minor revision suggestions.

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 2]
```diff
@@ -72,6 +72,72 @@ std::set<std::vector<element::Type>> jit_add_emitter::get_supported_precisions(c
...
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_clamp_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
+        OPENVINO_THROW("unsupported precision: " + exec_prc_.to_string());
+    }
```
[the remark this answers is not in the document]
Comment (tatyana-oliva) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 3]
```diff
@@ -72,6 +72,72 @@ std::set<std::vector<element::Type>> jit_add_emitter::get_supported_precisions(c
...
+void jit_clamp_emitter::register_table_entries() {
+    push_arg_entry_of("min", dnnl::impl::float2int(min), true);
+    push_arg_entry_of("max", dnnl::impl::float2int(max), true);
+}
+
+void jit_clamp_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done, thanks!

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: done, thanks

File: src/core/src/op/multiply.cpp (hunk 1 above)
Comment (eshoguli) [reply src/core/src/op/multiply.cpp]: rolled back, thanks

**Review by dmitry-gorokhov — APPROVED** [review]

**Review by praasz — APPROVED** [review]
